updatecli/updatecli.d/eks-addons.yaml

```diff
@@ -0,0 +1,145 @@
+    spec:
+      command: |
+        aws eks describe-addon-versions \
+          --kubernetes-version 1.29 \
+          --region us-east-2 \
+          --addon-name eks-pod-identity-agent \
+          --query 'addons[0].addonVersions[0].addonVersion' \
+          --output text
+      environments:
+          - name: PATH
+          - name: AWS_ACCESS_KEY_ID
+          - name: AWS_SECRET_ACCESS_KEY
+
+  getLatestAwsEbsCsiDriverVersion:
+    kind: shell
+    name: Retrieve the latest version of aws-ebs-csi-driver for Kubernetes 1.29
+    spec:
+      command: |
+        aws eks describe-addon-versions \
+          --kubernetes-version 1.29 \
+          --region us-east-2 \
+          --addon-name aws-ebs-csi-driver \
+          --query 'addons[0].addonVersions[0].addonVersion' \
+          --output text
+      environments:
+          - name: PATH
+          - name: AWS_ACCESS_KEY_ID
+          - name: AWS_SECRET_ACCESS_KEY
+
+  getLatestCoreDnsVersion:
+    kind: shell
+    name: Retrieve the latest version of coredns for Kubernetes 1.29
+    spec:
+      command: |
+        aws eks describe-addon-versions \
+          --kubernetes-version 1.29 \
+          --region us-east-2 \
+          --addon-name coredns \
+          --query 'addons[0].addonVersions[0].addonVersion' \
+          --output text
+      environments:
+          - name: PATH
+          - name: AWS_ACCESS_KEY_ID
+          - name: AWS_SECRET_ACCESS_KEY
+
+targets:
+  upgradeKubeProxyVersion:
+    name: Update the kube-proxy addon version in eks-cijenkinsio-agents-2.tf
+    kind: hcl
+    sourceid: getLatestKubeProxyVersion
+    spec:
+      file: eks-cijenkinsio-agents-2.tf
+      path: cluster_addons.kube-proxy.addon_version
+
+  upgradeVpcCniVersion:
+    name: Update the vpc-cni addon version in eks-cijenkinsio-agents-2.tf
+    kind: hcl
+    sourceid: getLatestVpcCniVersion
+    spec:
+      file: eks-cijenkinsio-agents-2.tf
+      path: module.cijenkinsio_agents_2.cluster_addons.vpc-cni.addon_version
+
+  upgradeEksPodIdentityAgentVersion:
+    name: Update the eks-pod-identity-agent addon version in eks-cijenkinsio-agents-2.tf
+    kind: hcl
+    sourceid: getLatestEksPodIdentityAgentVersion
+    spec:
+      file: eks-cijenkinsio-agents-2.tf
+      path: module.cijenkinsio_agents_2.cluster_addons.eks-pod-identity-agent.addon_version
+
+  upgradeAwsEbsCsiDriverVersion:
+    name: Update the aws-ebs-csi-driver addon version in eks-cijenkinsio-agents-2.tf
+    kind: hcl
+    sourceid: getLatestAwsEbsCsiDriverVersion
+    spec:
+      file: eks-cijenkinsio-agents-2.tf
+      path: module.cijenkinsio_agents_2.cluster_addons.aws-ebs-csi-driver.addon_version
+
+  upgradeCoreDnsVersion:
+    name: Update the coredns addon version in eks-cijenkinsio-agents-2.tf
+    kind: hcl
+    sourceid: getLatestCoreDnsVersion
+    spec:
+      file: eks-cijenkinsio-agents-2.tf
+      path: module.cijenkinsio_agents_2.cluster_addons.coredns.addon_version
+
+actions:
```

Comment:
As the actions (https://www.updatecli.io/docs/plugins/actions/github/) cannot be specified per targets, I suggest you split this manifest into multiple manifests